src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetector.java

```diff
@@ -128,6 +130,19 @@ public void addResult(Branch toBranch, List<GitMergeConflict> mergeConflicts,
     mergeResults.add(new MergeResult(toBranch, mergeConflicts, messages, files));
   }
 
+  public boolean isRelated(Branch toBranch)
+  {
+    String family = toBranchName.substring(toBranchName.lastIndexOf("/")).replaceAll("\\/([^\\.|-]*).*", "$1");
+    String toBranchFamily = toBranch.getDisplayId().substring(toBranch.getDisplayId().lastIndexOf("/")).replaceAll("\\/([^\\.|-]*).*", "$1");
```

Comment:
The naming is a little confusing here, but I suspect this is dealing with existing variable names?